Azure/azure-linux/tests/cron_command1.json

```
@@ -0,0 +1,44 @@
+{
+    "input": {
+        "sekoiaio": {
+            "intake": {
+                "dialect": "azure-linux",
+                "dialect_uuid": "162064f0-c594-455e-ac24-2d7129137688"
+            }
+        },
+        "message": "{ \"time\" : \"2019-06-27T14:50:01.0000000Z\",\"resourceId\" : \"/subscriptions/128ed5ce-4f50-4b5f-a3b0-08233b5a86b6/resourceGroups/demo.sekoia.io/providers/Microsoft.Compute/virtualMachines/LinuxRedhatDesktop\",\"properties\" : {\"ident\" : \"CROND\",\"pid\" : \"21188\",\"Ignore\" : \"syslog\",\"Facility\" : \"cron\",\"Severity\" : \"info\",\"EventTime\" : \"2019-06-27T14:50:01+0000\",\"SendingHost\" : \"localhost\",\"Msg\" : \"(root) CMD (/usr/lib64/sa/sa1 1 1)\",\"hostname\" : \"LinuxRedhatDesktop\",\"FluentdIngestTimestamp\" : \"2019-06-27T14:50:01Z\"},\"category\" : \"cron\",\"level\" : \"info\",\"operationName\" : \"LinuxSyslogEvent\"}"
+    },
+    "expected": {
+        "@timestamp": "2019-06-27T14:50:01.0000000Z",
+        "event": {
+            "outcome": "success"
+        },
+        "azure_linux": {
+            "message": "(root) CMD (/usr/lib64/sa/sa1 1 1)"
+        },
+        "log": {"hostname": "LinuxRedhatDesktop", "level": "info"},
+        "message": "{ \"time\" : \"2019-06-27T14:50:01.0000000Z\",\"resourceId\" : \"/subscriptions/128ed5ce-4f50-4b5f-a3b0-08233b5a86b6/resourceGroups/demo.sekoia.io/providers/Microsoft.Compute/virtualMachines/LinuxRedhatDesktop\",\"properties\" : {\"ident\" : \"CROND\",\"pid\" : \"21188\",\"Ignore\" : \"syslog\",\"Facility\" : \"cron\",\"Severity\" : \"info\",\"EventTime\" : \"2019-06-27T14:50:01+0000\",\"SendingHost\" : \"localhost\",\"Msg\" : \"(root) CMD (/usr/lib64/sa/sa1 1 1)\",\"hostname\" : \"LinuxRedhatDesktop\",\"FluentdIngestTimestamp\" : \"2019-06-27T14:50:01Z\"},\"category\" : \"cron\",\"level\" : \"info\",\"operationName\" : \"LinuxSyslogEvent\"}",
+        "process": {
+            "executable": "/usr/lib64/sa/sa1",
+            "command_line": "/usr/lib64/sa/sa1 1 1",
+            "pid": 21188
```

This process is not `/usr/lib64/sa/sa1`’s one, its the parent one (`CROND`’s one). I suggest to move this PID to `process.parent.pid` or to remove the information.

```suggestion
            "parent": {
                "pid": 21188
             }
```